ballerina/procedure_params.bal

```diff
@@ -31,11 +31,11 @@ public distinct class IntervalOutParameter {
     } external;
 }
 
-# Represents Inet OutParameter in `sql:ParameterizedCallQuery`.
+# Represents the Inet OutParameter in `sql:ParameterizedCallQuery`.
```

Comment:
`Inet OutParameter` or `InetOutParameter`?

Reply:
I believe it should be `Inet OutParameter` as in this context, we are referring to an OutParameter of type `Inet`